### ISSUE TITLE
Dungeon The Black Morass

### ISSUE THREAD

**Issue by notaslave** [Helios-Empire/Helios-Tracker#213]
Portals start spawning rapidly after killing the first boss, I would say a new one every 10 seconds or so. After we cleared rift 6 we were on rift 12 and swarmed with mobs in a minute or less.

Multiple mobs also stand and the portal they spawn at and cast 50-150 yards at medivh rather than running to him as they are supposed to.

**Comment by notaslave** [comment]
When we wiped at rift 13 with temporus up, medivh should have respawned with temporus still up and our progress reset to rift 13 with temporus still up as that's the last boss rift that spawned when we died

**Comment by HEGladios** [comment]
I made some changes, will be active once I restart the server, which I plan to do sometime tonight.

**Comment by HEGladios** [comment]
Changes are active, let me know what the results are.

**Comment by notaslave** [comment]
Will do, at the moment I'm the only one on so it might be a day or two unless everyone just randomly decides it's time to do stuff lol

**Comment by HEGladios** [comment]
I made some more changes and they are now active on the server.
- The portal timers have been fixed. (for real this time)
- Major improvements to rift adds, they won't instantly shoot lasers at Medivh.
- Script will no longer restart after the last boss has been killed. (for real this time)
- Related quests will be completed upon killing the last boss. Make sure you are close to Medivh when the last boss dies to ensure that you get quest credit.
